### DISCUSSION TITLE
Object has been destroyed 😅

### DISCUSSION THREAD

**Discussion by deeeiznc** [johnlindquist/kit#1512] [Errors]
Please report to our [GitHub Discussions](https://github.com/johnlindquist/kit/discussions/categories/errors)

## Kit 2.25.59Electron 32.0.1darwin 24.1.0Locale: en-US

    TypeError: Object has been destroyed
        at WebContents.disposable (file:///Applications/Kit.app/Contents/Resources/app.asar/node_modules/electron-context-menu/index.js:377:8)
        at Object.onceWrapper (node:events:634:26)
        at WebContents.emit (node:events:519:28)

**Comment by johnlindquist** [comment]
sorry, been testing windows/linux arm64 builds and tagging releases so the SDK downloads work correctly. They're not meant to be public yet.

**Comment by johnlindquist** [comment]
@deeeiznc Can you give this build a try?

https://github.com/script-kit/app/releases/tag/v2.26.4

**Reply by deeeiznc** [reply]
Successfully fixed, thank you!
